src/test_utils.rs

```diff
@@ -36,6 +36,30 @@ pub fn generate_validators(num_validators: u64) -> (Vec<SecretKey>, Arc<EpochInf
     (voting_sks, epoch_info)
 }
 
+pub fn generate_validators_with_leader_keys(
```

Comment:
this should probably be merged with the function above, just didn't want to break all references to `generate_validators()` in all tests